foundation-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,30 @@
 # Release Notes
 
+## Sept 3, 2021
+
+### iOS AEPCore 3.3.0
+
+* Added support for dispatch event rules consequence.
+
+### iOS AEPLifecycle 3.3.0
+
+* Lifecycle extension now dispatches two new events `applicationLaunch` and `applicationClose` which contain Mobile Lifecycle metrics in XDM format.
+* Fixed an issue where application upgrades were detected based on changes in `CFBundleShortVersionString` instead of `CFBundleVersion`.
+
+### Android Core 1.9.0
+
+* Enables dispatch rules for event data forwarding.
```

Comment:
Added support for dispatch event rules consequence.

Reply:
Praveen, you are suggesting to replace "Enables dispatch rules for event data forwarding." with "Added support for dispatch event rules consequence." or you want to add more details to it?

Reply:
Yes. Wanted to match the description for iOS and Android.